packages/lists/src/components/StaticList/StaticList.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { IStaticListProps } from './StaticList.types';
+
+function defaultRowRenderer<T>(item: T, index: number): JSX.Element {
+  return <li key={index}>{item}</li>;
+}
+
+class StaticList<T> extends React.PureComponent<IStaticListProps<T>> {
+  public render(): JSX.Element {
+    const { items = [], as: RootTag = 'ul', className, children = defaultRowRenderer } = this.props;
+
+    return <RootTag className={className}>{items.map(children)}</RootTag>;
```

Comment:
FYI - One case not covered by this list that the existing `List` handles is:

https://github.com/OfficeDev/office-ui-fabric-react/blob/c761a05177b59ce6e991d39235e9d0d672ed3ac7/packages/office-ui-fabric-react/src/components/List/List.test.tsx#L111

The above would need to be a for-loop to handle the sparse array case (which is rare).